src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,14 +1,37 @@
 import React from 'react';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { actions as filterActions } from '../../features/filter';
+import { Status } from '../../types/Status';
 
 export const TodoFilter: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const query = useAppSelector(state => state.filter.query);
+  const currentStatus = useAppSelector(state => state.filter.status);
+
+  const setStatus = (status: string) => {
+    dispatch(filterActions.setStatus(status as Status));
+  };
```

Comment:
set? use `handle` prefix for such functions.
also handle functions receive `event` by default
```suggestion
  const handleChangeStatus = (event: React.ChangeEvent<HTMLSelectElement>) => {
    dispatch(filterActions.setStatus(event.target.value));
  };
```
do it for all the cases!